controllers/subscriptions.go

```diff
@@ -76,6 +80,26 @@ var getSubscriptions = func(orgID string) types.SubscriptionsResponse {
 	var SubscriptionDetails []types.SubscriptionDetails
 	json.Unmarshal(body, &SubscriptionDetails)
 
+	/*
```

Comment:
Should probably delete this comment, unless there's a reason it's going into `master`

Reply:
Ok, I can take it out!